pkg/devfile/adapters/common/generic_test.go

```diff
@@ -300,8 +301,11 @@ func TestExecuteDevfileCommand(t *testing.T) {
 }
 
 func adapter(fakeExecClient ExecClient, commands []devfilev1.Command, cif func(command devfilev1.Command) (ComponentInfo, error)) *GenericAdapter {
-	data := &testingutil.TestDevfileData{}
-	_ = data.AddCommands(commands...)
+	data := func() data.DevfileData {
+		devfileData, _ := data.NewDevfileData(string(data.APIVersion200))
+		return devfileData
+	}()
+	_ = data.AddCommands(commands)
```

Comment:
Ignoring it here because 1) the function in which it is called doesn't have an error return type 2) it's in unit test and not in actual code 3) once `devfile/library` repo provides mocking mechanism (https://github.com/devfile/api/issues/383), we won't have to make function calls to achieve what we're doing here.